### ISSUE TITLE
[Bug] Cannot pass a comma-separated list of bootstrap nodes

### ISSUE THREAD

**Issue by CumpsD** [iron-fish/ironfish#771]
The following command will only use the first ip/port as a bootstrap node and will ignore the rest:

```
yarn start:once start -b xxx.xxx.xxx.xxx:9033,xxx.xxx.xxx.xxx:9033,xxx.xxx.xxx.xxx:9033,xxx.xxx.xxx.xxx:9033
```

This is because of the following code:

https://github.com/iron-fish/ironfish/blob/3cb1013d03757e62d0d3aff787a857f499bba512/ironfish-cli/src/commands/start.ts#L50

```
    bootstrap: flags.string({
      char: 'b',
      description: 'comma-separated addresses of bootstrap nodes to connect to',
      multiple: true,
    }),
```

The `multiple: true` in there means you can do:

``` 
yarn start:once start -b xxx.xxx.xxx.xxx:9033  -b xxx.xxx.xxx.xxx:9033  -b xxx.xxx.xxx.xxx:9033  -b xxx.xxx.xxx.xxx:9033
```

When you specify multiple like this, it works.

Do you want me to create a PR to change the wording?

**Comment by dguenther** [comment]
I opened a PR for this -- I think it worked at some point in the past, but we broke it at some point.

**Comment by deekerno** [comment]
Fixed through #786.
